style/calendar.less

```diff
@@ -0,0 +1,125 @@
+// Sample bootstrap theme for react-calendar
```

Comment:
Could we initially just `@import '../node_modules/react-calendar/...` instead of copying it into our codebase?